recipes/xvega/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "xvega" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "395d2c6890a4d9e28764ab39569663647138a7a9430f37c08fedd04c2cbd2dff" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xvega/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+  host:
+    - nlohmann_json
+    - xproperty
+    - xtl
+  run:
+    - nlohmann_json
+    - xproperty
+    - xtl
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/xvega  # [unix]
+    - test -d ${PREFIX}/include/xvega/base  # [unix]
+    - test -d ${PREFIX}/include/xvega/functions  # [unix]
+    - test -d ${PREFIX}/include/xvega/grammar  # [unix]
+    - test -d ${PREFIX}/include/xvega/utils  # [unix]
+    - test -f ${PREFIX}/include/xvega/xvega.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/base/xvega-base.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/functions/populate_data.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/grammar/marks.hpp  # [unix]
+    - test -f ${PREFIX}/include/xvega/utils/serialize.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xvega/xvegaConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xvega/xvegaConfigVersion.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\xvega\xvega.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\base\xvega-base.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\functions\populate_data.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\grammar\marks.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\xvega\utils\serialize.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xvegaConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xvegaConfigVersion.cmake (exit 0) else (exit 1)  # [win]
```

Comment:
Also, please ensure that you are not building static libraries

Reply:
(header only)